Library/ViewModels/PledgePaymentPlansViewModel.swift

```diff
@@ -7,30 +8,36 @@ public enum PledgePaymentPlansType: Equatable {
   case pledgeOverTime
 }
 
-public struct PledgePaymentPlansAndSelectionData: Equatable {
+public struct PledgePaymentPlansAndSelectionData {
   public var selectedPlan: PledgePaymentPlansType
+  public var paymentIncrements: [PledgePaymentIncrement]
+  public var project: Project
   /* TODO: add the necesary properties for the next states (PLOT Selected and Ineligible)
      - [MBL-1815](https://kickstarter.atlassian.net/browse/MBL-1815)
```

Comment:
Will we be able to remove this comment once this PR is merged?

Reply:
Sure, removed the line that is refer to the [MBL-1815]